modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorRuleHelper.java

```diff
@@ -119,6 +140,21 @@ protected boolean containsAction(
 			action -> Objects.equals(setBooleanPropertyAction, action));
 	}
 
+	protected boolean containsAction(
+		DDMFormRule ddmFormRule, String functionName, String ddmFormFieldName,
+		String defaultValue) {
+
+		String setStringPropertyAction = String.format(
+			"%s('%s', %s)", functionName, ddmFormFieldName, defaultValue);
+
+		List<String> actions = ddmFormRule.getActions();
+
+		Stream<String> stream = actions.stream();
+
+		return stream.anyMatch(
+			action -> Objects.equals(setStringPropertyAction, action));
+	}
```

Comment:
@rijema, I've realized that this new method has the same logic from the existing one, basically you just need a different type for the parameter `defaultValue`, am I right?

If so, can you try to use the existing method `containsAction` and change the type of the parameter `defaultValue` to something more generic, like `Object`?

I feel that you can reuse the existing method, can you please give it a try? :slightly_smiling_face: 

Thanks

Reply:
Changes made.